### ISSUE TITLE
Add a way to cache a document

### ISSUE THREAD

**Issue by pomdtr** [karelklima/workflowy#1]
Loading your workflowy list takes some time, so it would be nice to have a way to serialize/deserialize a workflowy document.

**Comment by karelklima** [comment]
Thank you for the issue. What use case did you have in mind?

Caching is possible with the current API already by instantiating the `Client`, fetching data manually (and caching them) and then instantiating the `Document`, exacly like here: https://github.com/karelklima/workflowy/blob/main/src/workflowy.ts#L40-L42

One problem with caching is that the WorkFlowy document will get out of sync rather quickly with the original. It is possible to write a synchronization logic, and I will likely do it in the near future, but the additional complexity may not be worth it.

Finally, it also depends whether you run a long-running script with this library, and then you can just leave the document in the memory, or whether you are building something like a CLI tool which runs end exits.

**Comment by pomdtr** [comment]
yeah this is linked with the design of sunbeam. Every page maps to a distinct process, so the data needs to be persisted between runs.

**Comment by karelklima** [comment]
Alright, thanks for the info. I saw that you solved that by using localStorage, I was thinking of doing the same within this library.
